articles/cosmos-db/sql-api-sdk-async-java.md

```diff
@@ -39,6 +39,31 @@ The SQL API Async Java SDK differs from the SQL API Java SDK by providing asynch
 
 ## Release notes
 
+### <a name="2.6.3"/>2.6.3
```

Comment:
Can this not follow the same pattern as this PR? 
https://github.com/MicrosoftDocs/azure-docs-pr/pull/88938

Reply:
For this PR:
Please update the dates and add the new entry at the end of this file.
Please update last update time.

The change to become like .Net for changelog can come in another PR.